tests/appsec/test_asm_standalone.py

```diff
@@ -78,11 +82,15 @@ def setup_no_appsec_upstream__no_attack__is_kept_with_priority_1__from_0(self):
 
     def test_no_appsec_upstream__no_attack__is_kept_with_priority_1__from_0(self):
         spans_checked = 0
-        for data, _, span in interfaces.library.get_spans(request=self.r):
+        for data, trace, span in interfaces.library.get_spans(request=self.r):
```

Comment:
I'm not very familiar with these tests, but couldn't be this just changes to `interfaces.library.get_root_spans(request=self.r)`? `REQUESTDOWNSTREAM_RESOURCE_PATTERN` wouldn't even be needed?
(might be addressed separately, since I don't think this PR is the place to refactor everything)

Reply:
`get_root_spans` gets spans which `parent_id == 0` but in these tests there are root spans with parent

Reply:
The `interfaces.library.get_root_spans` returns the spans with parentId = 0, in these tests we want the local root span (maybe it could be done with other approach)